types/api.d.ts

```diff
@@ -0,0 +1,121 @@
+import {Uri} from 'vscode';
+
+declare interface BasToolkit {
+    /**
+     * This an async wrapper for vscode extension that gets an extension by its full identifier and also wait until the target extension became active.
```

Comment:
its not a wrapper for vscode extension
its a wrapper for a vscode API.
Please link to the relevant API docs of vscode